platform/ios/src/MGLFaux3DUserLocationAnnotationView.m

```diff
@@ -291,13 +292,21 @@ - (void)drawDot
         if (accuracyRingSize > MGLUserLocationAnnotationDotSize + 15)
         {
             _accuracyRingLayer.hidden = NO;
+
+            id shouldNotAnimateAccuracyRingChange = (_oldHorizontalAccuracy == self.userLocation.location.horizontalAccuracy) ? (id)kCFBooleanTrue : (id)kCFBooleanFalse;
```

Comment:
Nit: always express variable names in the positive, for easier reasoning. How about `shouldDisableActions`, along with a comment?